### DISCUSSION TITLE
Account creating "issue"

### DISCUSSION THREAD

**Discussion by mrdadone** [Paymenter/Paymenter#444] [Ideas]
I have observed that many users attempt to log in to the  Pterodactyl panel for the first time using the same password they set in Paymenter. Could we implement a feature in which Paymenter creates the Pterodactyl account with the same password?

**Comment by Bansku-Dansku** [comment]
https://dashflo.net/docs/api/pterodactyl/v1/#req_802e27a56fe142c99db4106d8e8e8892 😎

**Comment by KeizerWilmer** [comment]
This is idea is a duplicate of https://github.com/Paymenter/Paymenter/discussions/377
